dotnet/src/webdriver/PrintOptions.cs

```diff
@@ -148,77 +134,91 @@ public void AddPageToPrint(int pageNumber)
         /// Adds a range of pages to be included in the document.
         /// </summary>
         /// <param name="pageRange">A string of the form "x-y" representing the page numbers to include.</param>
+        /// <exception cref="ArgumentException">
+        /// <para>If <paramref name="pageRange"/> is <see langword="null"/> or <see cref="string.Empty"/>.</para>
+        /// <para>-or-</para>
+        /// <para>If the requested <paramref name="pageRange"/> is already included.</para>
+        /// <para>-or-</para>
+        /// <para>If the requested <paramref name="pageRange"/> has multiple '-' separators.</para>
+        /// <para>-or-</para>
+        /// <para>If a bound value is neither empty nor a number.</para>
+        /// </exception>
+        /// <exception cref="ArgumentOutOfRangeException">
+        /// <para>If <paramref name="pageRange"/> has a negative lower bound.</para>
+        /// <para>-or-</para>
+        /// <para>If <paramref name="pageRange"/> has an upper bound less than the lower bound.</para>
+        /// </exception>
         public void AddPageRangeToPrint(string pageRange)
         {
             if (string.IsNullOrEmpty(pageRange))
             {
-                throw new ArgumentException("Page range cannot be null or the empty string");
+                throw new ArgumentException("Page range cannot be null or the empty string", nameof(pageRange));
             }
 
             if (this.pageRanges.Contains(pageRange))
             {
-                throw new ArgumentException("Cannot add the same page range twice");
+                throw new ArgumentException("Cannot add the same page range twice", nameof(pageRange));
             }
 
             string[] pageRangeParts = pageRange.Trim().Split('-');
             if (pageRangeParts.Length > 2)
             {
-                throw new ArgumentException("Page range cannot have multiple separators");
+                throw new ArgumentException("Page range cannot have multiple separators", nameof(pageRange));
             }
 
             int startPage = ParsePageRangePart(pageRangeParts[0], 1);
             if (startPage < 1)
             {
-                throw new ArgumentException("Start of a page range must be greater than or equal to 1");
+                throw new ArgumentOutOfRangeException(nameof(pageRange), "Start of a page range must be greater than or equal to 1");
```

Comment:
Some of these exceptions are better off as `ArgumentOutOfRangeException`s. Luckily this extends `ArgumentException` so there is no breaking change.